### ISSUE TITLE
Need to escape illegal characters in PRC notation

### ISSUE THREAD

**Issue by diafero** [H-uru/libhsplasma#15]
Some characters must be escaped when used in data or attributes in an XML file, such as the ampersand character. Otherwise, XML parsers will reject the file.

LibhPlasma methods that output prc files or views should change "&" to "&amp;", and methods that input prc files should change "&amp;" back to "&", and do the same for the other illegal characters.

(from https://code.google.com/p/libhsplasma/issues/detail?id=75)
#### Comment 1:

Only place I can think of where an & might occur is inside the raw hex data blocks for mipmaps or unsupported classes.

Could we avoid needing to escape by wrapping those in a <[CDATA[ ... ]]> block?
#### Comment 2:

```
<plDrawableSpans>
<plKey Name="Personal_District_psnlMYSTII_00000000_0Spans" Type="plDrawableSpans" Location="13;1" LocFlag="0x0000" ObjID="1" />
<Properties Flags="0x00000000" Criteria="0x00000000" RenderLevel="0x00000000" />
<Materials>
    <plKey Name="Top&Bottom" Type="hsGMaterial" Location="13;1" LocFlag="0x0000" ObjID="165" />
```

Basically, anywhere the age builder can enter a name, such as materials, textures, etc. But also technically, any manual change in the prp could do that.
CDATA blocks are a  possibility, but not as simple or safe as escaping when exporting, and de-escaping when importing.
